canvas/services/event.py

```diff
@@ -73,6 +74,14 @@ def set_featured(event):
 
 
 def add_question_set(event, category_id, difficulty, number_of_questions):
+    def begins_with_number(string):
+        # return any(char.isdigit() for char in string)
+        return string[0].isdigit()
+
+    def extract_1st_number(string):
+        # return re.search('[0-9]+', string).group()
+        return re.findall(r"\d+", string)[0]
```

Comment:
make this functions error proof so if there is not digit return something like -1 or 0